src/patternfly/components/Table/table.scss

```diff
@@ -390,10 +391,22 @@
 .pf-c-table__sort {
   // override pf-m-plain color
   .pf-c-button {
+    // stylelint-disable
     &.pf-m-plain {
       color: var(--pf-c-table__sort--c-button--Color);
     }
   }
+  // stylelint-enable
+
+  &.pf-m-wrap {
+    .pf-c-button {
+      display: flex;
```

Comment:
I'd put `whitespace` here as it doesn't require `.pf-c-table__sort--c-button-text`.

Reply:
@mcoker mentioned in slack: "then we could also be more specific about what we style with white-space - not that it hurts to style all of pf-c-button, but we may not want that at some point"